install_dependencies.sh

```diff
@@ -1,30 +1,99 @@
 #!/bin/bash
 
 
+help_install_dependencies="Usage:
+./install_dependencies.sh [os_name]         default OS is linux
+./install_dependencies.sh --help            displays help
+
+os_name can be:
+  - linux
+  - osx
+
+it installes the given dependencies:
```

Comment:
`installs` instead of `installes`, please also be careful with sentence capitalization.

Reply:
ohh sorry! my mistake :/